app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] having added the belongs to part with author you can remove this belongs_to relation.